src/modules/uORB/uORB_tests/uORBTest_UnitTest.cpp

```diff
@@ -597,7 +597,7 @@ int uORBTest::UnitTest::test_queue()
 		return test_fail("subscribe failed: %d", errno);
 	}
 
-	const int queue_size = 11;
+	const int queue_size = 16;
```

Comment:
Search for `ORB_QUEUE_LENGTH` in `msg/` and update them as well.

Reply:
I modified it because the test cannot be passed without modification, maybe we should provide an interface to get the real queue size.

Reply:
Manually setting all of them to a power of 2 (with code to catch and adjust any we miss) is fine by me.

Reply:
Yes, I agree. It's done.